packages/frontend/core/src/components/page-list/docs/page-list-header.tsx

```diff
@@ -69,7 +82,12 @@ export const PageListHeader = () => {
         onCreateEdgeless={e =>
           createEdgeless(isNewTabTrigger(e) ? 'new-tab' : true)
         }
-        onCreatePage={e => createPage(isNewTabTrigger(e) ? 'new-tab' : true)}
+        onCreatePage={e =>
+          createPage(
```

Comment:
how about make this default in usePageHelper.createPage?

Reply:
I made some changes in the latest commit, what do you think about that?